builder/js/ncf.js

```diff
@@ -939,12 +940,12 @@ $scope.onImportFileChange = function (fileEl) {
 
   // Check if a technique has not been changed, and if we can use reset function
   $scope.isUnchanged = function(technique) {
-    return (!$scope.suppressFlag && angular.equals(technique, $scope.originalTechnique));
+    return angular.equals(technique, $scope.originalTechnique);
   };
 
   // Check if a technique has been saved,
   $scope.isNotSaved = function() {
-    return $scope.originalTechnique !== undefined && $scope.originalTechnique.bundle_name === undefined;
+    return $scope.originalTechnique != undefined && $scope.originalTechnique.bundle_name === undefined;
```

Comment:
Why removing the second = in !==

Reply:
We do not only want to check if is `$scope.originalTechnique` is strictly different from undefined, we want to check if it is different from **null** too. I could write `($scope.originalTechnique !== undefined && $scope.originalTechnique !== undefined)` instead